src/index.ts

```diff
@@ -198,6 +206,14 @@ export class TopsortBanner extends LitElement {
         auction.category = {
           id: this.categoryId,
         };
+      } else if (this.categoryIds) {
```

Comment:
so this means that `category` has priority over `categoryIds` and `disjunctions` in the case the user would select more than one. The docs should reflect that.

Reply:
We don't support mixing `categoryIds` and `disjunctions` in the API correct? Only one should be set.
I'll update the docs

Reply:
Exactly. Only one of the options is allowed